packages/math-input/src/components/keypad/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,71 @@
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import * as React from "react";
+import "@testing-library/jest-dom";
+
+import Button from "../button";
+
+describe("<Button />", () => {
+    it("uses the aria label", () => {
+        // Arrange
+        render(
+            <Button onPress={jest.fn()} ariaLabel="Oranges">
+                <p />
+            </Button>,
+        );
+
+        // Assert
+        expect(
+            screen.getByRole("button", {name: "Oranges"}),
+        ).toBeInTheDocument();
+    });
+
+    it("handles callback on press", () => {
+        // Arrange
+        const mockPressCallback = jest.fn();
+        render(
+            <Button onPress={mockPressCallback} ariaLabel="Oranges">
+                <p />
+            </Button>,
+        );
+
+        // Assert
+        userEvent.click(screen.getByRole("button", {name: "Oranges"}));
+        expect(mockPressCallback).toHaveBeenCalled();
+    });
```

Comment:
```suggestion
        // Arrange
        const mockPressCallback = jest.fn();
        render(
            <Button onPress={mockPressCallback} ariaLabel="Oranges">
                <p />
            </Button>,
        );
        
        // Act
        userEvent.click(screen.getByRole("button", {name: "Oranges"}));

        // Assert
        expect(mockPressCallback).toHaveBeenCalled();
    });
```
I know we use Arrange/Act/Assert, I just can't ever remember what's what. I think this should be it?